pkg/cmd/edge_applications/publish/publish.go

```diff
@@ -89,6 +90,7 @@ func NewCobraCmd(publish *PublishCmd) *cobra.Command {
 	}
 	publishCmd.Flags().BoolP("help", "h", false, msg.EdgeApplicationsPublishFlagHelp)
 	publishCmd.Flags().StringVar(&Path, "path", "public", msg.EdgeApplicationPublishPathFlag)
+	publishCmd.Flags().StringSliceVarP(&Ignore, "ignore", "i", []string{}, msg.EdgeApplicationPublishIgnoreFlag)
```

Comment:
as a convention, we are only using shorthand for identifying flags (such as ID, UUID, etc...)

Reply:
Done! ✅